algorithmic_efficiency/workloads/librispeech_conformer/librispeech_pytorch/workload.py

```diff
@@ -114,24 +114,24 @@ def _build_input_queue(self,
                          global_batch_size: int,
                          num_batches: Optional[int] = None,
                          repeat_final_dataset: bool = False):
-    del data_rng
     del num_batches
     del repeat_final_dataset
     train = False
-
+    eval_train = False
```

Comment:
I don't think we need this (it doesn't actually save us any lines of code, just a single python compare)?

Reply:
The reason I included this is because we can create a subset of ds only after ds is created? Otherwise, we would have to replicate line 132 inside each of the if-branches? Alternatively, we can avoid renaming the original function argument by giving a new name to the `split` inside the if-conditions?

Reply:
I just mean, can you delete `eval_train` and change L133 back to `if split == 'eval_train':`

Reply:
Yes, we cannot do it because we are changing the function argument `split` inside the if condition :)

Reply:
Ahhh I totally missed that, my bad! I think it can be confusing to change the value of the variable we're conditioning on in an if/elif, so instead of having eval_train can we make a new var called tfds_split that we set in each branch?